internal/action/generate_test.go

```diff
@@ -150,6 +150,32 @@ func TestGenerate(t *testing.T) {
 		assert.NoError(t, act.Generate(gptest.CliCtxWithFlags(ctx, t, map[string]string{"force": "true", "xkcd": "true", "print": "true", "lang": "en"}, "foobar", "baz")))
 		buf.Reset()
 	})
+
+	// generate --force foobar 24 w/ autoclip and output redirection
+	t.Run("generate --force foobar 24", func(t *testing.T) {
+		ov := act.cfg.AutoClip
+		defer func() {
+			act.cfg.AutoClip = ov
+		}()
+		act.cfg.AutoClip = true
+		ctx := ctxutil.WithTerminal(ctx, false)
+		assert.NoError(t, act.Generate(gptest.CliCtxWithFlags(ctx, t, map[string]string{"force": "true"}, "foobar", "24")))
+		assert.Contains(t, buf.String(), "Not printing secrets by default")
```

Comment:
Are we printing this on `StdOut`? If so we might want to consider doing that on `StdErr` instead in a future PR.

Reply:
Yes, I think this goes to stdout right now.